src/OpenConext/Profile/Tests/Value/EmailAddressTest.php

```diff
@@ -51,14 +51,13 @@ public function it_doesnt_accept_non_strings_as_email($nonString)
     /**
      * @test
      * @group Value
+     *
+     * @expectedException \OpenConext\Profile\Exception\InvalidArgumentException
      */
-    public function email_address_may_contain_mailto()
+    public function email_address_may_not_contain_mailto()
     {
         $emailAddressWithMailto = 'mailto:mail@domain.invalid';
-
-        $emailAddress = new EmailAddress($emailAddressWithMailto);
-
-        $this->assertEquals($emailAddressWithMailto, (string) $emailAddress);
+        new EmailAddress($emailAddressWithMailto);
```

Comment:
Could you test this case along with some other cases like empty string, missing @, etc.